diffedit.md

```diff
@@ -0,0 +1,752 @@
+# DiffEdit
+
+In this blog post, we will be demonstrating how to use the DiffEdit technique described in [this paper](https://arxiv.org/abs/2210.11427) (TODO: should this be more of a citation format?), to use a diffusion model to modify just one part of an existing image using simple text prompts. DiffEdit utilises the diffusion model which is used to predict where noise is in an image, typically as a way of generating images using text prompts. In this blog post we assume you have basic knowledge of Stable Diffusion and how it works, as this is the diffusion model that we will be using throughout our examples. If you want to read more about diffusion models, then check out our other blog post about it here [TODO]().
+
+We will work through an example where we use the DiffEdit technique to modify an image of a horse into an image of a zebra.
+
+Many diffusion models already can be used in an image to image pipeline which can take in an original image and tweak it based on a new prompt, which may at first seem similar to the DiffEdit technique. Image to image is fairly limited however, as all parts of the image will be changed by this process, you cannot limit the modifications of the image to a specific area.
```

Comment:
It's better but maybe could be improved a bit further. n.b. I'm only labouring it because it's the start of the post where you need to build trust with the reader that you're not going to leave them baffled. 

Here's GPT4's suggestion -

> In this post, we'll walk through an example of using the DiffEdit technique to transform an image of a horse into a zebra.
> 
> Many existing diffusion models can be utilized in an image-to-image pipeline. This process takes an original image and modifies it based on a new prompt, which may seem similar to the DiffEdit technique at first glance. However, the image-to-image method has its limitations. For instance, it alters all parts of the image, and there's no way to restrict the modifications to a specific area.
> 
> The DiffEdit paper introduces a novel approach. It suggests using the noise prediction provided by a diffusion model to automatically generate a mask. > This mask can then be used to confine the area of the image that can be altered. 
> 
> In the following sections, we'll first explore how to automatically generate this mask and discuss a few techniques to enhance this process. Subsequently, we'll demonstrate how to use this mask to edit only a specific part of our original image.

Reply:
Scratch that I was reviewing the old version.